.travis.yml

```diff
@@ -24,7 +24,9 @@ cache:
   - target
 
 before_script:
-  - rustup component add rustfmt-preview
+  - rustup toolchain install nightly-2019-12-20
```

Comment:
should use a more recent toolchain. maybe same as the one used by tikv? https://github.com/tikv/tikv/blob/master/rust-toolchain